hawkmoth/docstring.py

```diff
@@ -142,7 +142,14 @@ def get_docstring(self, transform=None):
 
         text = Docstring._nest(text, self._indent)
 
-        args = ', '.join(self._args) if self._args is not None else None
+        name = self._get_decl_name()
+
+        ttype = self._ttype
+
+        args = ''
```

Comment:
Feels like this should remain None if there are no args. If it "leaks" to `_fmt.format()` I think it'll be more obvious there's a bug somewhere?

Reply:
Ahah, used in the 2nd commit.

Reply:
Sheesh github landed this comment in a silly place. This was related to `name = self._get_decl_name()`.

Reply:
So I may be misreading this sequence of comments, but I'm guessing the 1st comment is still about `args = ''`? If so, note that `self._args` may still be `None`, `args` on the other hand is the processed argument string we want to use in the final line and that has to be an actual string. Did I misunderstand anything?

By the way, there could be a point in handling `self._args` in the derived classes. I could create a method to abstract the argument stitching. Same for the type spacer thing. It felt a bit verbose in comparison, but might make it nicer overall.

Reply:
I meant that it's a (theoretical) functional change to set `args = ''`.  It was set to `None` before, but it's fine when `self._fmt` does not contain `{args}`.
Anyway, not a big deal, I think I'll just merge this as-is.
Thanks for doing this!

Reply:
Ah, I see now. Well, I didn't fix the `name` and `ttype` being defined one commit ahead, but alright.
Thanks for the review!